app/code/Magento/Sales/Model/Order.php

```diff
@@ -213,6 +215,11 @@ class Order extends AbstractModel implements EntityInterface, OrderInterface
     protected $_currencyFactory;
 
     /**
+     * @var \Magento\Eav\Model\Config
+     */
+    private $_eavConfig;
```

Comment:
If this is a new variable you do not need the `_` at the beginning.

Reply:
It isn't a new variable, just a missing phpdoc.

Reply:
In version `2.2-develop` of this file there is no such variable. https://github.com/magento/magento2/blob/2.2-develop/app/code/Magento/Sales/Model/Order.php It seems be have been added to `develop` in you original PR. I wonder if it is a left over from when the file was created. I will see what I can find out.

Reply:
@dmanners I remember, variable was already there but never declared as a property.

https://github.com/magento/magento2/blob/2.2-develop/app/code/Magento/Sales/Model/Order.php#L338

Reply:
Yup seems to be. My guess is we can remove the property and the mapping but leave the injection for BiC reasons. But I will dig a bit deeper to make sure.

Reply:
Sure, ping me if you need any changes done in this PR.

Reply:
Yup so this looks like a left over from the M1 approach of doing things. I cannot find a usage in this class and in the history it was last changed 4 years ago. I would suggest you can remove the new property, remove the assignment in the constructor `$this->_eav` and add the tag `@SuppressWarnings(PHPMD.UnusedFormalParameter)` to the constructor phpdocs.